src/reconcile/rec_row.c

```diff
@@ -90,6 +91,8 @@ __rec_cell_build_leaf_key(
 
     btree = S2BT(session);
     key = &r->k;
+    ovfl_ps.records = WT_STAT_NONE;
```

Comment:
We can define ovfl_ps inside __wt_rec_cell_build_ovfl and only pass in the size.